Packages/xmgrace/Lib/xmgrace.py

```diff
@@ -3080,9 +3080,14 @@ def add_ellipse(self, x1, y1, x2, y2,
         self.nellipse = len(self.Box)
         return
 
+    def clear():
+        """ clears all graphs """
+        for i, g in enumerate(self.Graph):
+            for j in range(g.nset):
+                self("kill G%i.S%i" % (i, j))
```

Comment:
??

/home/buildbot/buildbot/uvcdat-garant-linux-release/source/Packages/xmgrace/Lib/xmgrace.py:3085:31: F821 undefined name 'self'
        for i, g in enumerate(self.Graph):
                              ^
/home/buildbot/buildbot/uvcdat-garant-linux-release/source/Packages/xmgrace/Lib/xmgrace.py:3087:17: F821 undefined name 'self'
                self("kill G%i.S%i" % (i, j))
                ^
2       F821 undefined name 'self'